README.md

```diff
@@ -16,18 +16,27 @@ For now, that's just [Django](https://www.djangoproject.com/) and [MongoDB](http
 
 ## Contact Information
 
-The point of creating CoGS is to have a solution for the MRDC game server that is as portable, maintainable, and transparent as possible. We learned our lesson from [Dredd](https://github.com/uiuc-mrdc/game-server).
+The point of creating CoGS is to have a solution for the MRDC game server that is as portable, maintainable, and transparent as possible. We learned our lesson; we want CoGS to last for years and to be accessible to people who weren't here when we wrote it.
 
-In case this documentation is insufficient, or there are other problems with this system, you can contact [lpulley](https://lpulley.me/).
+In case this documentation is insufficient, or there are other problems with this system, you can contact [lpulley](https://github.com/lpulley/).
 
 ## Detailed Documentation
 
 ### Running and Stopping
 
 Run `docker-compose up -d` in the root of this repo to start all of the CoGS services in the background, or to update the containers if they are already running.
 
-To stop the containers, run `docker-compose stop`. To restart them use `docker-compose start`. To stop the containers and delete them, use `docker-compose down` (note that while this does delete volumes, it does **not** delete host files, so for example the database data in `./data-db` will persist). The `down` subcommand is useful for starting from *almost* scratch, such that only data linked to the host filesystem persists, but every service is completely recycled.
+To stop the containers, run `docker-compose stop`. To restart them use `docker-compose start`. To stop the containers and delete them, use `docker-compose down` (note that while this does delete volume connections, it does **not** delete host files created by volumes, so for example the database data in `./data-db` will persist). The `down` subcommand is useful for starting from *almost* scratch, such that only data in volumes persists, but every service is completely recycled.
```

Comment:
```suggestion
To stop the containers, run `docker-compose stop`. To restart them use `docker-compose start`. To stop the containers and delete them, use `docker-compose down` (note that while this does delete volume connections, it does **not** delete host files created by volumes or images, so for example the database data in `./data-db` will persist). The `down` subcommand is useful for starting from *almost* scratch, such that only data in volumes persists, but every service is completely recycled. Running `docker-compose up -d` will start them anew, but does not rebuild the images. To rebuild from the `docker-compose.yml` and `Dockerfile` you must remove the image with `docker rmi <imagename>` or `docker-compose --rmi all`
```

Reply:
Really good suggestion. Not sure why I didn't think to mention this. Thanks!